v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/templates/MongoDbToBigQuery.java

```diff
@@ -77,11 +74,37 @@ public static boolean run(Options options) {
                 .withCollection(options.getCollection()))
         .apply(
             "Transform to TableRow",
-            MapElements.via(
-                new SimpleFunction<Document, TableRow>() {
-                  @Override
-                  public TableRow apply(Document document) {
-                    return MongoDbUtils.generateTableRow(document, userOption);
+            ParDo.of(
+                new DoFn<Document, TableRow>() {
+
+                  @ProcessElement
+                  public void process(ProcessContext c) {
```

Comment:
This code seems to be duplicated in MongoDbToBigQueryCdc. I suggest moving it to the Utils class or something like this, e.g. convert the input Document to Map here, and the input String to Map in MongoDbToBigQueryCdc, and then convert Map->Row via a common utils method.

Reply:
Hi Nick, This method serves different purpose and takes and returns different parameters from that of the MongoDbToBigQueryCdc template.